controllers/offer.js

```diff
@@ -20,11 +20,10 @@ const getOfferById = async (req, res) => {
 }
 
 const createOffer = async (req, res) => {
-  const { id: authorId } = req.params
-  const { role: authorRole } = req.user
+  const { id: authorId, role: authorRole } = req.user
   const offer = req.body
```

Comment:
shuldn't it be 'data' like in other cases?

Reply:
true

Reply:
+